Support for ARM64

Opening a parallel issue to https://github.com/overhangio/tutor/issues/510. I'm currently able to build on an M1 MacBook by adding a few extra build requirements and environment variables:

```diff
diff --git a/tutormfe/templates/mfe/build/mfe/Dockerfile b/tutormfe/templates/mfe/build/mfe/Dockerfile
index 726d9ca..b996eea 100644
--- a/tutormfe/templates/mfe/build/mfe/Dockerfile
+++ b/tutormfe/templates/mfe/build/mfe/Dockerfile
@@ -3,7 +3,18 @@ FROM docker.io/node:12-bullseye-slim AS base
 RUN apt update \
   && apt install -y git \
     # required for cwebp-bin
-    gcc git libgl1 libxi6 make
+    gcc git libgl1 libxi6 make \
+    # additionally required for gifsicle, mozjpeg, and optipng
+    autoconf libtool pkg-config zlib1g-dev \
+    # additionally required for node-sass
+    python g++
+
+# Required for building optipng on M1
+ENV CPPFLAGS=-DPNG_ARM_NEON_OPT=0
+
+# Required for avoiding failed frontend-app-learning build due to missing binary on M1
+# (breaks pact tests)
+ENV PACT_SKIP_BINARY_INSTALL=true
 
 RUN mkdir -p /openedx/app /openedx/env
 WORKDIR /openedx/app
```

Unfortunately, using `PACT_SKIP_BINARY_INSTALL` is a hack that will break any Pact tests, so it's not really suitable to merge in. I don't see much alternative until there's an ARM binary for Pact though. Relevant issue: pact-foundation/pact-js-core#347

I am running into an issue with the following:
```bash
FROM docker.io/node:12-bullseye-slim AS base

RUN apt update \
  && apt install -y git \
    # required for cwebp-bin
    gcc git libgl1 libxi6 make \
    # additionally required for gifsicle, mozjpeg, and optipng
    autoconf libtool pkg-config zlib1g-dev \
    # additionally required for node-sass
    python g++
    
# Required for avoiding failed frontend-app-learning build due to missing binary on M1
# (breaks pact tests)
ENV PACT_SKIP_BINARY_INSTALL=true

RUN mkdir -p /openedx/app /openedx/env
```

However, when I run `tutor images build mfe` I get the following error:

```bash
#71 42.45 ERROR in ./src/index.scss
#71 42.45 Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
#71 42.45 ModuleBuildError: Module build failed (from ./node_modules/image-webpack-loader/index.js):
#71 42.45 Error: Cannot find module 'gifsicle'
#71 42.45 Require stack:
```

Per [suggestions](https://stackoverflow.com/questions/65147246/cannot-find-module-gifsicle-for-loading-image-webpack) on the Internet, tried updating my Dockerfile to the following to install `dh-autoreconf`, but still run into the same error:
```
RUN apt update \
  && apt install -y git \
    # required for cwebp-bin
    gcc git libgl1 libxi6 make \
    # additionally required for gifsicle, mozjpeg, and optipng
    autoconf libtool pkg-config zlib1g-dev \
    # additionally required for node-sass
    python g++
    
RUN apt-get install dh-autoreconf
```

--- a/tutormfe/templates/mfe/build/mfe/Dockerfile
+++ b/tutormfe/templates/mfe/build/mfe/Dockerfile
@@ -3,7 +3,11 @@ FROM docker.io/node:12-bullseye-slim AS base
 RUN apt update \
   && apt install -y git \
     # required for cwebp-bin
-    gcc git libgl1 libxi6 make
+    gcc git libgl1 libxi6 make \
+    # additionally required for gifsicle, mozjpeg, and optipng (on arm)
+    autoconf libtool pkg-config zlib1g-dev \
+    # additionally required for node-sass (on arm)
+    python g++
 
 RUN mkdir -p /openedx/app /openedx/env
 WORKDIR /openedx/app
@@ -32,10 +36,15 @@ COPY --from=i18n /openedx/i18n/i18n-merge.js /openedx/i18n/i18n-merge.js
 RUN /openedx/i18n/i18n-merge.js /openedx/app/src/i18n/messages /openedx/i18n/{{ app["name"] }} /openedx/app/src/i18n/messages
 ######## {{ app["name"] }} (dev)
 FROM base AS {{ app["name"] }}-dev
+
 COPY --from={{ app["name"] }}-src /openedx/app/package.json /openedx/app/package.json
 COPY --from={{ app["name"] }}-src /openedx/app/package-lock.json /openedx/app/package-lock.json
 ARG NPM_REGISTRY=https://registry.npmjs.org/
 {{ patch("mfe-dockerfile-pre-npm-install") }}
+{# Required for building optipng on M1 #}
+ENV CPPFLAGS=-DPNG_ARM_NEON_OPT=0
+{# We define this environment variable to bypass an issue with the installation of pact https://github.com/pact-foundation/pact-js-core/issues/264 #}
+ENV PACT_SKIP_BINARY_INSTALL=true
 RUN npm install --no-audit --no-fund --registry=$NPM_REGISTRY \
   && rm -rf ~/.npm
 {{ patch("mfe-dockerfile-post-npm-install") }}

